provision/rosa-cross-dc/Taskfile.yaml

```diff
@@ -576,20 +576,17 @@ tasks:
         - ROSA_CLUSTER_NAME
     vars:
       RETENTION: '{{.RETENTION | default "168h"}}'
-      JAEGER_VERSION: '{{.JAEGER_VERSION | default "0.58.0"}}'
-    env:
-      KB_RETENTION: '{{.RETENTION}}'
+      JAEGER_VERSION: '{{.JAEGER_VERSION | default "3.1.2"}}'
     cmds:
       - >
         KUBECONFIG="{{.ISPN_DIR}}/.task/kubecfg/{{.ROSA_CLUSTER_NAME}}"
         helm upgrade --install jaeger jaegertracing/jaeger --version {{.JAEGER_VERSION}} -n monitoring -f "{{.ROOT_DIR}}/../minikube/jaeger/values.yaml"
-      - envsubst < "{{.ROOT_DIR}}/../minikube/jaeger/deployment-patch.yaml" > .task/jaeger-{{.ROSA_CLUSTER_NAME}}-patchfile.yaml
-      - >
-        KUBECONFIG="{{.ISPN_DIR}}/.task/kubecfg/{{.ROSA_CLUSTER_NAME}}"
-        oc patch deployment jaeger -n monitoring --patch-file .task/jaeger-{{.ROSA_CLUSTER_NAME}}-patchfile.yaml
+        --set allInOne.podSecurityContext.runAsUser=$(cat .task/monitoring-uids-{{.ROSA_CLUSTER_NAME}})
+        --set allInOne.podSecurityContext.runAsGroup=$(cat .task/monitoring-uids-{{.ROSA_CLUSTER_NAME}})
+        --set allInOne.podSecurityContext.fsGroup=$(cat .task/monitoring-uids-{{.ROSA_CLUSTER_NAME}})
+        --set allInOne.extraEnv[0].value={{.RETENTION}}
```

Comment:
I tried this locally and I got docker.io rate limit error. We can add 
```
--set allInOne.image.registry=quay.io
```
to load the image from quay. This can then be applied also to other jaeger helm chart installations.

Reply:
Thanks, added it to the values.yaml instead.